base_classes/NXobject.nxdl.xml

```diff
@@ -30,5 +30,164 @@
+			must not be reassigned to a different real-world object. It is typical for
+			there to be some mechanism to resolve an identifier, obtaining metadata
+			about the object. Identifiers for which some guarantees exist regarding
+			this resolution process are called persistent identifiers.
+			Persistent identifiers are also known as PIDs.
+		</doc>
+		<attribute name="type" type="NX_CHAR">
+			<doc>The type of identifier used.</doc>
+			<enumeration>
+				<item value="ARK">
+					<doc>
+						Archival Resource Key.
+						
+						An ARK is a Uniform Resource Identifier (URI) designed to support long-term access to a variety of information objects.
+						
+						Syntax: https://NMA/ark:/NAAN/Name[Qualifier]. Brackets indicate optional elements.
+						Example: https://example.org/ark:/12345/abcde
+					</doc>
+				</item>
+				<item value="DOI">
+					<doc>
+						Digital Object Identifier.
+						
+						A DOI is a unique alphanumeric string used to identify digital content. It consists of a prefix and a suffix, separated by a slash.
+						
+						Syntax: 10.XXXX/XXXXXX
+						Example: 10.1107/S1600576714027575
+					</doc>
+				</item>
+				<item value="Hdl">
+					<doc>
+						A handle is a unique identifier that consists of a prefix indicating the naming authority and a suffix representing the local name 
+						of a resource, cts. 
+						
+						A handle is a unique identifier used to facilitate the identification and management of digital objects. It is composed of a prefix that 
+						indicates the naming authority and a suffix that specifies the resource's local name. 
+						
+						This refers specifically to an ID in the Handle system operated by the Corporation for National Research Initiatives (CNRI).
+						
+						Syntax: hdl:prefix/identifier
+						Example: hdl:123456789/abc123
+					</doc>
+				</item>
+				<item value="IGSN">
+					<doc>
+						International Generic Sample Number.
+						
+						The IGSN is a unique identifier assigned to a specific sample or specimen in the context of scientific research.
+						
+						Syntax: https://igsn.org/{IGSN-ID}
+						Example: https://igsn.org/IGSN:XYZ123456
+					</doc>
+				</item>
+				<item value="ISNI">
+					<doc>
+						ISNI is an ISO standard to uniquely identify individuals and organizations involved in creative work, including pseudonyms
+						and other public personas.
+						
+						An ISNI is made up of 16 digits, the last character being a check character. The check character may be either a decimal digit
+						or the character “X”. 
+						
+						Syntax: https://isni.org/isni/{ISNI-ID}
+						Example: https://isni.org/isni/0000000121032683
+					</doc>
+				</item>
+				<item value="ISSN">
+					<doc>
+						International Standard Serial Number
+						
+						An ISSN is an 8-digit unique identifier used to distinguish a serial publication, whether in print or electronic form.
+						
+						Syntax: ISSN XXXX-XXXX
```

Comment:
As above,  I don't think the prefix `ISSN ` in the syntax adds anything useful as the `type` attribute already identifies the value as an ISSN.

Also (borrowing from Wikipedia) the syntax might be better expressed as `NNNN-NNNC`

where N is in the set {0,1,2,...,9}, a decimal digit character, and C is in {0,1,2,...,9,X};